autoload/denops_shared_server/runtray.vim

```diff
@@ -41,6 +41,11 @@ function! denops_shared_server#runtray#uninstall() abort
   call delete(s:script_file)
 endfunction
 
+function! denops_shared_server#runtray#restart() abort
+  call denops_shared_server#util#info('restart the service')
+  call denops_shared_server#runtray#_execute_script_command('restart')
```

Comment:
The `restart` command was added in [runtray v1.4.3](https://github.com/Milly/runtray-ps/releases/tag/v1.4.3). We need to change `s:script_download_url`.

Reply:
Done.